Ports/SDL2/patches/0002-Avoid-accumulation-errors-in-resampler.patch

```diff
@@ -0,0 +1,23 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Tim Ledbetter <timledbetter@gmail.com>
+Date: Wed, 9 Aug 2023 23:08:44 +0100
```

Comment:
Wrong authorship information. Please take the upstream patch from [here](https://github.com/libsdl-org/SDL/commit/78f97108f9dad22db6708f28322561eb9828fc36.patch) as-is (after running it through `./package.sh dev` to update the styling).

Reply:
I didn't realize github could generate patch files like that. That's better, thanks :^)